runestone/fitb/__init__.py

```diff
@@ -1 +1,4 @@
+# *********
+# |docname|
+# *********
```

Comment:
In my opinion, including stuff like this just creates noise in the documentation.  Additional headers that add nothing.

I wish we could come up with a metric for what is useful and what makes it harder to navigate and learn what you want to learn.

Reply:
That's a good point. How about ignoring `__init__.py` in Sphinx if it's empty? (I don't know of an automatic way to do this, but I can manually add these excludes for now.)

I would also like to come up with a documentation style guide to help us this about things like this. Here's my [current list](https://codechat.readthedocs.io/en/master/docs/style_guide.cpp.html) -- what would you add/change?

Reply:
I went ahead and added ignores for `__init__.py`.

Reply:
I implemented this with a short script in the BookServer project. If you like that. I'll do the same here.

Reply:
For now, I removed the additions above. Eventually, we can use the script to programmatically ignore the noise.